cloudferrylib/os/actions/prepare_networks.py

```diff
@@ -31,6 +31,13 @@ def run(self, info=None, **kwargs):
         keep_ip = self.cfg.migrate.keep_ip
 
         instances = info_compute[utl.INSTANCES_TYPE]
+        # disable DHCP in all subnets
+        subnets = network_resource.get_subnets()
+        for snet in subnets:
+            if snet['tenant_name'] == self.cfg.dst.tenant:
+                network_resource. \
```

Comment:
This is actually not required (at least for pep8 and flake8 with default configuration). Looks like this is only openstack style requirement, which we can skip.